eva/catalog/schema_utils.py

```diff
@@ -40,15 +40,15 @@ def get_petastorm_column(df_column):
         if column_type == ColumnType.INTEGER:
             petastorm_column = UnischemaField(
                 column_name,
-                np.int32,
+                np.int64,
                 (),
                 ScalarCodec(IntegerType()),
                 column_is_nullable,
             )
         elif column_type == ColumnType.FLOAT:
             petastorm_column = UnischemaField(
                 column_name,
-                np.float64,
+                np.float32,
```

Comment:
What is the motivation of these changes?

Reply:
I changed it to python's default `int` and `float` types.